configstack/running_module.go

```diff
@@ -286,9 +286,11 @@ type MultiError struct {
 func (err MultiError) Error() string {
```

Comment:
I didn't realize we had two `MultiError` structs, one in `configstack` and one in `errors`. We should consolidate on just one of them; probably the latter.

Reply:
I have refactored accordingly.